k8s/readme.md

```
@@ -26,8 +26,85 @@ Ensure that you have the following installed on your system:
+   kubectl -n kube-system rollout restart deployment coredns
+   ```
+
+   4. **Deploy the Proxy Service:**
+
+   Use the `deploy.sh` script to deploy the proxy service in the Kubernetes cluster:
+
+   ```bash
+   ./scripts/deploy.sh
+   ```
+
+### Example 1: Without Proxy 
```

Markdownlint

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)